src/averageAges.js

```diff
@@ -14,12 +14,17 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = 0) {
+  const age = century !== 0
+    ? people
+      .filter((person) => Math.ceil(person.died / 100) === century)
+      .filter((person) => person.sex === 'm')
+      .map((person) => +`${person.died - person.born}`)
+    : people
+      .filter((person) => person.sex === 'm')
+      .map((person) => +`${person.died - person.born}`);
```

Comment:
the same about mapping
you shouldn't write it twice

also, why do you transform the value you get into a string if you want to manipulate it as a number (inside reduce) next?